app/assets/styles/_z-index.scss

```diff
@@ -0,0 +1,56 @@
+$z-index: (
```

Comment:
Optional: could clean the values here, but keeping their order the same.

Reply:
Good suggestion 👍 
Done!